src/xrUICore/CMakeLists.txt

```diff
@@ -1,90 +1,179 @@
+    "InteractiveBackground/UI_IB_Static.h"
+    "Lines/UILine.cpp"
+    "Lines/UILine.h"
+    "Lines/UILines.cpp"
+    "Lines/UILines.h"
+    "Lines/UISubLine.cpp"
+    "Lines/UISubLine.h"
+    "Lines/uilinestd.h"
+    "ListBox/UIListBox.cpp"
+    "ListBox/UIListBox.h"
+    "ListBox/UIListBoxItem.cpp"
+    "ListBox/UIListBoxItem.h"
+    "ListBox/UIListBoxItemMsgChain.cpp"
+    "ListBox/UIListBoxItemMsgChain.h"
+    "ListBox/UIListBox_script.cpp"
+    "ListWnd/UIListItem.cpp"
+    "ListWnd/UIListItem.h"
+    "ListWnd/UIListItemEx.cpp"
+    "ListWnd/UIListItemEx.h"
+    "ListWnd/UIListWnd.cpp"
+    "ListWnd/UIListWnd.h"
+    "ListWnd/UIListWnd_inline.h"
+    "ListWnd/UIListWnd_script.cpp"
+    "MessageBox/UIMessageBox.cpp"
+    "MessageBox/UIMessageBox.h"
+    "MessageBox/UIMessageBox_script.cpp"
+    "Options/UIOptionsItem.cpp"
+    "Options/UIOptionsItem.h"
+    "Options/UIOptionsManager.cpp"
+    "Options/UIOptionsManager.h"
+    "Options/UIOptionsManagerScript.cpp"
+    "Options/UIOptionsManagerScript.h"
+    "ProgressBar/UIDoubleProgressBar.cpp"
+    "ProgressBar/UIDoubleProgressBar.h"
+    "ProgressBar/UIProgressBar.cpp"
+    "ProgressBar/UIProgressBar.h"
+    "ProgressBar/UIProgressBar_script.cpp"
+    "ProgressBar/UIProgressShape.cpp"
+    "ProgressBar/UIProgressShape.h"
+    "PropertiesBox/UIPropertiesBox.cpp"
+    "PropertiesBox/UIPropertiesBox.h"
+    "PropertiesBox/UIPropertiesBox_script.cpp"
+    "ScrollBar/UIFixedScrollBar.cpp"
+    "ScrollBar/UIFixedScrollBar.h"
+    "ScrollBar/UIScrollBar.cpp"
+    "ScrollBar/UIScrollBar.h"
+    "ScrollBar/UIScrollBox.cpp"
+    "ScrollBar/UIScrollBox.h"
+    "ScrollView/UIScrollView.cpp"
+    "ScrollView/UIScrollView.h"
+    "SpinBox/UICustomSpin.cpp"
+    "SpinBox/UICustomSpin.h"
+    "SpinBox/UISpinNum.cpp"
+    "SpinBox/UISpinNum.h"
+    "SpinBox/UISpinText.cpp"
+    "SpinBox/UISpinText.h"
+    "Static/UIAnimatedStatic.cpp"
+    "Static/UIAnimatedStatic.h"
+    "Static/UILanimController.cpp"
+    "Static/UILanimController.h"
+    "Static/UIStatic.cpp"
+    "Static/UIStatic.h"
+    "Static/UIStaticItem.cpp"
+    "Static/UIStaticItem.h"
+    "Static/UIStatic_script.cpp"
+    "TabControl/UITabButton.cpp"
+    "TabControl/UITabButton.h"
+    "TabControl/UITabControl.cpp"
+    "TabControl/UITabControl.h"
+    "TabControl/UITabControl_script.cpp"
+    "TrackBar/UITrackBar.cpp"
+    "TrackBar/UITrackBar.h"
+    "Windows/UIFrameLineWnd.cpp"
+    "Windows/UIFrameLineWnd.h"
+    "Windows/UIFrameWindow.cpp"
+    "Windows/UIFrameWindow.h"
+    "Windows/UITextFrameLineWnd.cpp"
+    "Windows/UITextFrameLineWnd.h"
+    "Windows/UIWindow.cpp"
+    "Windows/UIWindow.h"
+    "Windows/UIWindow_script.cpp"
+    "XML/UITextureMaster.cpp"
+    "XML/UITextureMaster.h"
+    "XML/UIXmlInitBase.cpp"
+    "XML/UIXmlInitBase.h"
+    "XML/xrUIXmlParser.cpp"
+    "XML/xrUIXmlParser.h"
+)
 
-######################################################
+group_sources(SRC_FILES)
 
-source_group("arrow" FILES ${ARROW})
```

Comment:
Source groups removed? This is regression. No way.

Reply:
group_sources(SRC_FILES) does the same thing.

Reply:
Aaah, that's how it works.. Sorry :D